backend/src/entity/testEnt.ts

```diff
@@ -0,0 +1,15 @@
+import { Entity, Column, PrimaryGeneratedColumn } from "typeorm"
+
+@Entity()
+export class User {
+    @PrimaryGeneratedColumn()
+    id: number|null
```

Comment:
1) should be spaces between words even if it works
2) is it possible to have nullable primary key?